### ISSUE TITLE
Suggestion: Include information on where and how to get required files for essentia

### ISSUE THREAD

**Issue by gargolito** [mgoltzsche/beets-autogenre#3]
Getting these support files was difficult to locate. I use linux so that's what I downloaded and all worked. The file names are different so I just changed the location of the svm models and executable files to live in (I don't like putting things in /usr/lib that don't need to be there) ~/.config/beets/essentia/ and modified the config file to reflect that.

SVM Models: https://essentia.upf.edu/svm_models/
essentia binaries: https://essentia.upf.edu/extractors/

**Comment by mgoltzsche** [comment]
Good point. Here is a script to download them: https://github.com/mgoltzsche/essentia-container/blob/v0.1.0/Dockerfile#L42-L44

Alternatively, you can use the [`ghcr.io/mgoltzsche/beets-plugins:0.18.2`](https://github.com/mgoltzsche/beets-container) docker/Linux container (or a newer version) which has the Essentia models and a corresponding default configuration preinstalled.

Though, I agree that should be documented within this project's readme.
